modules/installation-guide/nav.adoc

```diff
@@ -10,28 +10,21 @@
 ** xref:using-chectl-to-configure-the-checluster-custom-resource-during-installation.adoc[]
 ** xref:using-the-cli-to-configure-the-checluster-custom-resource.adoc[]
 ** xref:checluster-custom-resource-fields-reference.adoc[]
-
-
 * xref:installing-che.adoc[]
-
-
 ** xref:installing-che-in-cloud.adoc[]
-
 *** xref:installing-che-on-openshift-4-using-operatorhub.adoc[]
 *** xref:installing-che-on-openshift-4-using-cli.adoc[]
-*** xref:installing-che-on-openshift-3-using-the-operator.adoc[]
-*** xref:installing-che-on-kubespray.adoc[]
-*** xref:installing-che-on-aws.adoc[]
-*** xref:installing-che-on-google-cloud-platform.adoc[]
-*** xref:installing-che-on-microsoft-azure.adoc[]
-
+// *** xref:installing-che-on-openshift-3-using-the-operator.adoc[] // REMOVE
+// *** xref:installing-che-on-kubespray.adoc[] // UNMAINTAINED
+// *** xref:installing-che-on-aws.adoc[] // UNMAINTAINED
```

Comment:
ROSA / AWS is important use case. We need this doc for CRW (but not sure if Che too).

Reply:
Here, the kops tool is used to install Kubernetes. -> OT for downstream